compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/PlatformWindowContext.desktop.kt

```diff
@@ -60,18 +66,16 @@ internal class PlatformWindowContext {
         _windowContainer = windowContainer
     }
 
-    fun setContainerSize(size: Size) {
-        _windowInfo.containerSize = IntSize(
-            width = size.width.roundToInt(),
-            height = size.height.roundToInt()
-        )
+    fun setContainerSizeFromComponent(component: Component) {
+        _windowInfo.containerSize = component.sizeInPx.roundToIntSize()
+        _windowInfo.containerDpSize = component.size.asDpSize()
```

Comment:
Can't we have a single source of truth for the size info? (since either of values can be derived from another one)

Reply:
Not yet - it's a common code. I've already suggested Googlers to fix it in AOSP, but most likely we'll be requried to contribute it ourselves.
Anyway, yes, I'm sharing your concern and already on it. But it won't be done in this PR